chabud/datapipe.py

```diff
@@ -137,6 +137,8 @@ def __init__(
             "https://huggingface.co/datasets/chabud-team/chabud-extra/resolve/main/california_0.hdf5",
             "https://huggingface.co/datasets/chabud-team/chabud-extra/resolve/main/california_1.hdf5",
             "https://huggingface.co/datasets/chabud-team/chabud-extra/resolve/main/california_2.hdf5",
+            "https://huggingface.co/datasets/chabud-team/chabud-extra/resolve/main/california_3.hdf5",
+            "https://huggingface.co/datasets/chabud-team/chabud-extra/resolve/main/california_4.hdf5",
```

Comment:
@weiji14 we can ignore the `california_*.hdf5` files for now, as the dataset is currently imbalanced. We can add them back once we implement the mixup & cutmix augmentations.

Reply:
Ok, I'll comment those lines out for now.